docker-compose/db-only-migrate.docker-compose.yaml

```diff
@@ -1,4 +1,4 @@
-# This file contains only the pgsql service definition, and is used during
+# This file contains the pgsql service definitions, and is used during
```

Comment:
Maybe database instead of pgsql? _Technically_ pgsql refers specifically to the main SG database deployment, and codeintel-db is a separate postgres deployment.